### PR TITLE
Update Spring Boot 2.1.11.RELEASE and Spring Cloud Greenwich.SR4 #3047

### PR DESCRIPTION
**Pull request by mteodori**
see Activiti/Activiti#3047

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -1,154 +1,110 @@
...
+    <dependency>
+      <groupId>org.springframework.cloud</groupId>
+      <artifactId>spring-cloud-starter-stream-rabbit</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.activiti.cloud.connector</groupId>
+      <artifactId>activiti-cloud-starter-connector</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>junit</groupId>
```
Comment (zoltanpalfi) [inline pom.xml]: Can be removed, redundant to `spring-boot-test`

File: pom.xml [hunk 2]
```diff
@@ -1,154 +1,110 @@
...
+      <artifactId>junit</artifactId>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.springframework.boot</groupId>
+      <artifactId>spring-boot-test</artifactId>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.assertj</groupId>
```
Comment (zoltanpalfi) [inline pom.xml]: Can be removed, redundant to `spring-boot-test`

File: pom.xml [hunk 3]
```diff
@@ -1,154 +1,110 @@
...
+      <artifactId>assertj-core</artifactId>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.mockito</groupId>
```
Comment (zoltanpalfi) [inline pom.xml]: Can be removed, redundant to `spring-boot-test`

File: pom.xml (hunk 2 above)
Comment (mteodori) [reply pom.xml]: done

File: pom.xml (hunk 1 above)
Comment (mteodori) [reply pom.xml]: done

File: pom.xml (hunk 3 above)
Comment (mteodori) [reply pom.xml]: done

**Pull request merged**